extensions/rich_text_components/Image/directives/ImageDirective.js

```diff
@@ -35,10 +35,11 @@ oppia.directive('oppiaNoninteractiveImage', [
         $scope.filepath = HtmlEscaperService.escapedJsonToObj(
           $attrs.filepathWithValue);
         $scope.imageUrl = '';
```

Comment:
Here imageUrl is '', in ImageClickInput it is null, we should be consistent.

Reply:
Done!